dashboard/app/models/unit.rb

```diff
@@ -36,6 +36,15 @@
 require 'cdo/shared_constants/curriculum/shared_course_constants'
 require 'ruby-progressbar'
 
+# These explicit requires are necessary because this model is required by the
+# script_preload initializer, before autoloaded constants are available.
```

Comment:
I went back and forth on whether I wanted to include a comment like this in every file, or just for the highest-visibility changes. My inclination is to put one everywhere, but that started feeling excessively verbose.

What do y'all think?